cms/templates/index.html

```diff
@@ -338,16 +338,18 @@ <h3 class="course-title">${course_info['display_name']}</h3>
 
       % if libraries_enabled:
       <ul id="course-index-tabs">
-        <li class="courses-tab active"><a>${_("Courses")}</a></li>
+        <li class="courses-tab active"><a>${_("Online Courses")}</a></li>
+        <li class="archive-tab"><a>${_("Archive")}</a></li>
         <li class="libraries-tab"><a>${_("Libraries")}</a></li>
       </ul>
       % endif
 
-      %if len(courses) > 0 or optimization_enabled:
+      %if (len(courses) > 0 or optimization_enabled) and (len(courses) != len([course_info for course_info in courses if course_info['archived'] == True])):
       <div class="courses courses-tab active">
         <ul class="list-courses">
           %for course_info in sorted(courses, key=lambda s: s['display_name'].lower() if s['display_name'] is not None else ''):
-          <li class="course-item" data-course-key="${course_info['course_key']}">
+          % if course_info['archived'] == False:
```

Comment:
You don't need to use == here, it is not a good practice to compare booleans like that, it's better to just write 'if not course_info['archived']'